blogArticles/nested/folder/hello-world.mdx

```diff
@@ -0,0 +1,12 @@
+import Content from 'components/Content/Content';
+import Head from 'components/head';
+import HeroBanner from 'components/HeroBanner/HeroBanner';
+
+export const meta = {
+  publishDate: '2018-05-10T12:00:00Z',
+  title: 'Hello, World!',
+};
+
+# Hello, world!
+
+Lorem ipsum dolor sit amet.
```

Comment:
We won't have a need to prepare for nested articles, which should let you get rid of the `path-utils` file you made and just leverage `pathname` in the `getInitialProps` for blog.

As an aside, as it stands, how do I route to any of these articles? I was trying, but not succeeding.

Reply:
Got it. I removed nested structure in ``blogArticles``

I made some changes to ``[Article].js``, mainly to use ``getStaticPaths`` and ``getStaticProps`` so I think it should work properly now.